pom.xml

```diff
@@ -48,7 +48,11 @@
       <url>file://${project.basedir}/maven-repo/</url>
     </repository>
   </distributionManagement>
-
+  
+  <prerequisites>
+    <maven>3.0.5</maven>
```

Comment:
The nbm-maven-plugin (which was updated to version 4.1) requires Maven version 3.0.5 or higher.